graspologic/types.py

```diff
@@ -5,14 +5,23 @@
 This module includes common graspologic type hint declarations.
 """
 
+import sys
 from typing import Union
 
 import networkx as nx
 import numpy as np
 import scipy.sparse as sp
 
+if sys.version_info >= (3, 9):
```

Comment:
the mypy/sys.version_info insanity deserves a comment

Reply:
something like:

```python
# mic ... not dropped, so much as thrown
```